src/css/components/modal.less

```diff
@@ -109,3 +119,99 @@
     }
   }
 }
+
+.modal.dialog {
+  color: @modal-dialog-text-color;
+  font-size: @modal-font-size;
+
+  .modal-header {
```

Comment:
Yep, you're right, but `modal-header` is the proper bootstrap class name, thus we should keep those redundant "namespace" instead of creating our own set of classes.